src/components/sidebar/sidebar-header/sidebar-header.style.js

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+import sidebarHeaderClassicStyle from './sidebar-header-classic.style';
+import baseTheme from '../../../style/themes/base';
+
+const SidebarHeaderStyle = styled.div`
+  background-color: ${({ theme }) => theme.colors.white};
+  font-size: 24px;
```

Comment:
Is this not just because there is a title in the design example (ie. a composed example) - shouldn't the text size be the same as it was?

Reply:
That's right, SidebarHeader should be only a wrapper. The content should be flexible. Fixed